src/enums/routes.ts

```diff
@@ -13,7 +13,5 @@ export enum Routes {
 
   // Digital Credential Routes for Pilot
   DIGITAL_CREDENTIALS = 'digital-credentials',
-  DOWNLOAD_WALLET = 'download-wallet',
-  REGISTER_WALLET = 'register-wallet',
-  ISSUE_CREDENTIALS = 'issue-credentials'
+  ISSUE = 'issue'
```

Comment:
Is this still a pilot? If not then please update line 14.

Could you come up with a less generic name for this route please? :)

Reply:
Thanks for changing the key name, but please also update the value (route name).

Reply:
The path is currently `/digitalCredentials/issue`. Would it be more descriptive to change it to `digitalCredentials/issue-credential`? Just seems like a long path name.

Reply:
Oh, I missed the fact that it's a child route... because it's declared like all other routes in the enum.

It makes sense to have 2 child routes ("" and "issue") but I'd like to prevent this confusion again. Ideas?

Reply:
Sounds good. Go for it.

Reply:
Looks good. Resolving.